_the_law/the-rules.md

```diff
@@ -1,129 +1,4 @@
-* Kicking suspends the offender from the room they were kicked in. Suspension time
-  [depends on the amount of times the offender was kicked before](http://meta.stackexchange.com/questions/229905/239226#239226).
-* Issues with messages can be escalated to the entire chat by using the flag mechanism
-  Note: every active user that has more than 10k reputation will be bothered with, and be able to act on
-  the flagged message. Do not treat flags lightly.
-* Flagging is done by clicking the flag icon (on the left when you hover over a message), or
-  choosing "flag as spam/offensive" from the message menu.
-* If [enough users](http://meta.stackoverflow.com/questions/100414/100439#100439) will deem the flag to be
-  valid, the offender will be automatically suspended from all rooms on the chat.
-* You can escalate issues to site moderators by choosing "flag for moderator" option in the message
-  menu (on the bottom, without the flag icon next to it). **Use this as a last resort.**
-* **Abuse of the flag system will invoke the wrath of the entire chat.
-  We will report any case of suspected abuse to the site moderators.
-  This includes, but is not limited to, any form of revenge flagging.
-  If you're not 100% sure your flag is valid, *don't flag*.**
-
-### History
-
-* The [transcript of the room](http://chat.stackoverflow.com/transcript/10) is public.
-* You can (try to) search the transcript using the search box in the upper right corner, or
-  by visiting [the search page](http://chat.stackoverflow.com/search?room=10&q=).
-* You can also [browse the history of starred messages](http://chat.stackoverflow.com/rooms/info/10/loungec?tab=stars).
-
-### Culture
-
-* Abuse of acronyms is rampant in these parts. We have compiled [a list of most commonly used ones](/the-law/acronyms).
-* We aim to have around 10 owners (picked usually from most active regulars), and possibly
-  distributed around the world for more timezone coverage.
-* There were some not-really-rules-more-guidelines set up [in the past](http://chat.stackoverflow.com/transcript/10?m=2264159#2264159)
-  about ownership. They might or might not still apply.
-* Kyrostat is not dead.
-
-### Keyboard shortcuts
-
-Some of the standard shortcuts supported by the chat. You can get more by installing
-[chat extensions](https://github.com/rchern/StackExchangeScripts/).
-
-Shortcut                 | Function
------------------------- | --------
-<kbd>Enter</kbd>         | Send message, save edit
-<kbd>Shift+Enter</kbd>   | Add a newline to the message
-<kbd>↑</kbd>             | Enter edit mode (if message is single line and you have editable messages), go to previous message
-<kbd>↓</kbd>             | In edit mode, go to next message
-<kbd>Escape</kbd>        | Exit edit mode
-<kbd>Ctrl+K</kbd>        | Toggle fixed font
-<kbd>Tab</kbd>           | After typing `@`, choose between autocompleted nicks
+# COCK
```

Comment:
I would like to take this opportunity to discuss the immediate transition of power over the Lounge to one Elim Garak, a loyal citizen of the Cardassian Union and the right hand of the head of the Obsidian Order.